contracts/contracts/token/OUSD.sol

```diff
@@ -34,33 +34,21 @@ contract OUSD is Initializable, InitializableERC20Detailed, Governable {
         uint256 rebasingCreditsPerToken
     );
 
-    // MAX_SUPPLY is chosen to guarantee applied changes to _totalSupply in
-    // changeSupply(_newTotalSupply) deviate from the value provided in
-    // _newTotalSupply by < 1
-    uint256 private constant MAX_SUPPLY = ~uint128(0); // (2^128) - 1
+    enum RebaseOptions { NotSet, OptOut, OptIn }
 
+    uint256 private constant MAX_SUPPLY = ~uint128(0); // (2^128) - 1
```

Comment:
I have verified with remix that the value set in MAX_SUPPLY really matches the comment. (Sadly, google's built in calculator lies about this - had to drop down to `bc` on the terminal)

<img width="379" alt="image" src="https://user-images.githubusercontent.com/837/103388945-eed18800-4ad9-11eb-9a5b-408790d5d103.png">